neutone_sdk/audio.py

```diff
@@ -81,20 +109,23 @@ def get_default_audio_samples() -> List[AudioSample]:
     log.info(
         "Using default sample... Please consider using your own audio samples by overriding the get_audio_samples method"
     )
-    wave_inst, sr_inst = torchaudio.load(
+    wave_inst, sr_inst = sf.read(
         io.BytesIO(
             pkgutil.get_data(__package__, "assets/default_samples/sample_inst.mp3")
         ),
-        format="mp3",
+        always_2d=True,
     )
-    wave_music, sr_music = torchaudio.load(
+    wave_music, sr_music = sf.read(
         io.BytesIO(
             pkgutil.get_data(__package__, "assets/default_samples/sample_music.mp3")
         ),
-        format="mp3",
+        always_2d=True,
     )
 
-    return [AudioSample(wave_inst, sr_inst), AudioSample(wave_music, sr_music)]
+    return [
+        AudioSample(tr.from_numpy(wave_inst.T), sr_inst),
+        AudioSample(tr.from_numpy(wave_music.T), sr_music),
```

Comment:
To prevent having to do this annoying process every time and making sure the transpose is correct, let's just add a class method to the AudioSample class called `from_file_path(path: str)` or something like that, that handles all this for you, and then you can just reuse that method yourself as well. This will let users easily make an AudioSample from .wav or .mp3 files.

Reply:
Never mind we already have the static method above that does this.

Reply:
We don't have this, right? It only works from base64 strings.